spec/lib/tasks/publishing_api_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "publishing_api.rake" do
+  it "should publish special routes" do
+    publishing_api_url = "http://publishing-api.dev.gov.uk".freeze
+    content_id = "bce40c1f-2259-4404-b275-8c5e04afef34".freeze
+
+    Rake.application = Rake::Application.new
+    Rake.application.rake_require("lib/tasks/publishing_api", [Rails.root.to_s], [])
+    Rake::Task.define_task(:environment)
+
+    put_content_stub = stub_request(:put, "#{publishing_api_url}/v2/content/#{content_id}")
+    publish_content_stub = stub_request(:post, "#{publishing_api_url}/v2/content/#{content_id}/publish")
```

Comment:
There's some GdsApi helpers that are commonly used for this type of stub. If you include the GdsApi::TestHelpers::PublishingApi module you can replace with:

```
put_content_stub = stub_publishing_api_put_content(content_id, {})
publish_content_stub = stub_publishing_api_publish(content_id, {})
```

Reply:
✨